projects/hslayers/README.md

```diff
@@ -13,10 +13,14 @@ HSLayers-NG is a library which extends OpenLayers 6 functionality by providing a
 
 ## Demo
 
-Check out the examples to get an idea:  
+Check out the examples and live projects to get an idea:  
 * http://ng.hslayers.org/examples/
-* http://opentransportnet.eu/create-maps
-* http://sdi4apps.eu/spoi
```

Comment:
I'd like to object that sdi4apps.eu/spoi (or https://app.hslayers.org/spoi/) is a live project ;-)

Reply:
But two active projects per library is probably enough, I admit.

Reply:
> I'd like to object that sdi4apps.eu/spoi (or https://app.hslayers.org/spoi/) is a live project ;-)

You're right, I will add it back.